### PR TITLE
README.md update

### PR DESCRIPTION
**Pull request by Sor4nex**
fix: README.md was extended

### CODE REVIEW

**Comment by Lesh79** [comment]
I think need some docs and pictures from work.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,84 @@
-# PySATL-IGraph_CPD-Module
+# PySATL-CPD-Module
 
-Dependencies installation
+PLACE FOR BADGES
```
Comment (alexdtat) [inline README.md]: About badges: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge

Also you can see them in the neighboring projects:
1) https://github.com/toxakaz/EM-algo/blob/main/README.md?plain=1
2) https://github.com/SE-Processor-Fuzzing/control-hazard-analyzer/blob/main/README.md?plain=1

**Review by alexdtat — APPROVED** [review]
Overall LGTM

File: README.md [hunk 2]
```diff
@@ -1,25 +1,88 @@
-# PySATL-IGraph_CPD-Module
+# PySATL-CPD-Module
 
-Dependencies installation
+<a href="https://github.com/Lesh79/PySATL-CPD-Module/actions"><img alt="Lint" src="https://github.com/Lesh79/PySATL-CPD-Module/actions/workflows/build-and-test.yaml/badge.svg"></a>
```
Comment (alexdtat) [inline README.md]:
```suggestion
```

This workflow seems outdated and should be removed.

**Review by alexdtat — APPROVED** [review]
Please delete the outdated workflow's badge.

File: README.md [hunk 3]
```diff
@@ -1,25 +1,88 @@
...
+<a href="https://github.com/Lesh79/PySATL-CPD-Module/actions"><img alt="Test" src="https://github.com/Lesh79/PySATL-CPD-Module/actions/workflows/check.yaml/badge.svg"></a>
+<a href="https://github.com/psf/black"><img alt="Code style: black" src="https://img.shields.io/badge/code%20style-black-000000.svg"></a>
+<a href="https://github.com/Lesh79/PySATL-CPD-Module/blob/main/LICENSE"><img alt="License: MIT" src="https://black.readthedocs.io/en/stable/_static/license.svg"></a>
 
-```shell
+**Change point detection module** (*abbreviated CPD module*) is a module, designed for detecting anomalies in time series data, which refer to significant deviations from expected patterns or trends. Anomalies can indicate unusual events or changes in a system, making them crucial for monitoring and analysis in various fields such as finance, healthcare, and network security.
+
```
Comment (alexdtat) [inline README.md:8]:
```suggestion

At the moment, the module implements the following CPD algorithms:
* Bayesian algorithm
* Density based algorithms:
    * KLIEP
    * RuLSIF
* Graph algorithm

```

Almost fortot the very important part.

Comment (alexdtat) [reply README.md:8]: It should be a nested unordered list, please fix formatting if this suggestion will display incorrectly.

**Review by vkutuev — APPROVED** [review]

**Pull request merged**